CHANGELOG.md

```diff
@@ -20,9 +20,11 @@ security issues.
 
 ### IMPROVEMENTS
 
+- `[mempool]` Remove redundant error return in `(*Mempool).Update`
```

Comment:
Please add the same entry (using unclog, see my comment on that) again in `breaking-changes` directory, as this is changing a public interface.